### PR TITLE
Added options to select apps and compare counties

### PR DESCRIPTION
**Pull request by Alig1493**
#17 @banjtheman this is my take on tckling the budget comparison app for different counties.

### CODE REVIEW

File: apps/county_compare/views.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    # select county
+    counties = os.listdir(STATES_FOLDER + state)
+    first_county = st.selectbox("Select First County", counties)
+    second_county = st.selectbox("Select Second County", counties)
+
+    if first_county == second_county:
+        st.error("Please select different counties!!")
+        st.stop()
+
```
Comment (banjtheman) [inline apps/county_compare/views.py:28]: Allow user to pick from different State

File: main_st.py [hunk 2]
```diff
@@ -31,233 +23,13 @@ def show_menu():
...
-        text_width, text_height = canvas.textsize(line, font=font)
-
-        x_pos = int((image_width - text_width) / 2)
-        y_pos = starter + pad
-        canvas.text((x_pos, y_pos), line, font=font, fill=text_color)
-        pad += text_height + 5
-
-    dst = get_concat_v_cut(img, bar_chart)
-    st.image(dst)
+    selection = st.sidebar.radio("Go to", list(PAGES.keys()))
```
Comment (banjtheman) [inline main_st.py]: make a selectbox instead

**Comment by banjtheman** [comment]
Yes this is great, left two minor comments to address

**Comment by Alig1493** [comment]
Noted. I'm working on them and I'll update.

**Comment by Alig1493** [comment]
@banjtheman please review.

**Pull request merged**